### ISSUE TITLE
Dark Theme support is problematic

### ISSUE THREAD

**Issue by sunjam** [otetard/ownpad#90]
![doesnt-work](https://user-images.githubusercontent.com/1787238/62571640-b414e300-b846-11e9-81b2-2e261ae93c9e.png)

Text fields show up as black on black in the new Dark Theme, available in Nextcloud under Accessibility

**Comment by otetard** [comment]
This issue was fixed in 41cc470e624048b503ace6c5cea3e098306d48f7 (not yet released)
